test/scanner/Tests/ScanConditionalsSpec.hs

```diff
@@ -14,13 +14,53 @@ import           Front.Token                    ( TokenType(..) )
 spec :: Spec
 spec = do
   it "scans inline conditionals" $ do
-    pendingWith "help needed"
+    let input = "if a { b }"
+        output =
+          [ TIf
+          , TId "a" -- TODO(hans): What happened to TThen?
```

Comment:
Great question. I think when it saw `{`, it (correctly?) decided not to insert the additional block.